core/src/components/input/input.ionic.scss

```diff
@@ -15,12 +18,26 @@
   min-height: 48px;
 }
 
+// Input Bottom Content
+// ----------------------------------------------------------------
+
+.input-bottom {
+  @include padding-horizontal(0, 0);
+}
+
```

Comment:
While I was implementing the valid state, I noticed that some styles were missing from the helper text.